openedx/features/course_experience/utils.py

```diff
@@ -186,7 +188,9 @@ def recurse_mark_auth_denial(block):
         'discussion',
         'drag-and-drop-v2',
         'poll',
-        'word_cloud'
+        'word_cloud',
+        'lti',
+        'lti_consumer',
```

Comment:
We were previously excluding lti modules, so that they would never be marked as `scored`. This includes them.

It's not very scalable to other types... But it's the pattern of this whole class.

Just to double-check, `lti` is the type of the old LTI xmodule, right? I know `lti_consumer` is the new xblock version.

Reply:
Yeah, I wonder why we're filtering at all...